app/src/main/res/layout/fragment_main.xml

```diff
@@ -55,12 +55,82 @@
                     android:text="@string/no_profile"
                     android:textSize="14sp" />
         </RelativeLayout>
-
-        <androidx.recyclerview.widget.RecyclerView
-                android:id="@+id/rv"
+        <LinearLayout
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginTop="70dp" />
+                android:layout_height="match_parent"
+                android:layout_marginTop="70dp"
+                android:orientation="vertical">
+
+            <RelativeLayout
```

Comment:
I think this relative layout can be removed. Try to avoid nesting of views as much as possible for performance reasons.